ModelicaTest/Tables/CombiTable2Ds.mo

```diff
@@ -1095,4 +1095,92 @@ double mydummyfunc(double dummy_in) {
         points={{-59,-10},{-50,-10},{-50,4},{-42,4}}, color={0,0,127}));
     annotation (experiment(StartTime=0, StopTime=60));
   end Test32;
+  
+  model OneSidedDerivative2D "Test of one sided derivatives in 2D-tables"
```

Comment:
As for the example model:

* Please add ticket number to the model description (as already done for e.g. Test32 or Test33).
* Please add ModelicaTest/Resources/Reference/ModelicaTest/Tables/CombiTable2Ds/OneSidedDerivative2D/comparisonSignals.txt with the list of reference variables as required by all example and test models.
* And we also need to have it for CombiTable2Dv as well.

Reply:
Have added ticket number, vector-variant, and comparison signals.
The vector-variant may be a bit odd, since it is based on the scalar variant but I merged cases 1,2,4; and kept 3 separate (since 3 got the same result using a different matrix and interpolation method).
If that is too weird I would reorder the cases in both of them.